book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -0,0 +1,201 @@
+# Run a Wallaroo Application in Docker
+
+In this section, we're going to run an example Wallaroo application in Docker. By the time you are finished, you'll have validated that your Docker environment is set up and working correctly. If you haven't already completed the Docker setup [instructions](book/getting-started/docker-setup.md), please do so before continuing.
+
+There's a couple Wallaroo support applications that you'll be interacting with for the first time:
```

Comment:
"You will be interacting with a few Wallaroo auxiliary applications for the first time:"